code/controllers/subsystem/research.dm

```diff
@@ -7,7 +7,7 @@ SUBSYSTEM_DEF(research)
 	//TECHWEB STATIC
 	var/list/techweb_nodes = list() //associative id = node datum
 	var/list/techweb_designs = list() //associative id = node datum
-	var/list/datum/design/item_to_design = list() //typepath = list of design datums
+	var/list/list/datum/design/item_to_design = list() //typepath = list of design datums
```

Comment:
I didn't know you could make typed lists of typed lists.

Reply:
I didn't either it's a bit scuffed. basically every time you go "down" a list we remove one preceding /list from the type.

Reply:
yo dawg i heard you liked lists so i put a list in your list to track your lists